### ISSUE TITLE
Investigate loss of gene to phenotype associations in zebrafish

### ISSUE THREAD

**Issue by kshefchek** [monarch-initiative/dipper#498]
Our dev dataset is missing most zebrafish gene to phenotype associations.  The first place to look will be the mappings to the ZP ontology.

Production: 51200
Beta: 110

The ZFIN ingest should fail if there are issues with ZP lookups.

**Comment by kshefchek** [comment]
cc @drseb

**Comment by drseb** [comment]
What is ingested? Where is the ontology downloaded, where are the annotations taken from?

**Comment by kshefchek** [comment]
The ZP mapping is from the Charite pipeline, we pull this file:

http://compbio.charite.de/hudson/job/zp-owl-new/lastSuccessfulBuild/artifact/zp.annot_sourceinfo

and map the 6 part EQ annotation from the ZFIN data to this file. We also output a "missing zp" file for each run here, the output from the latest run is here:
https://data.monarchinitiative.org/dev/missing_zps.txt

**Comment by kshefchek** [comment]
Since there has been a large drop I suspect there may have a change in the order of fields, either from ZFIN for the ZP map file.  If nothing has changed on the ZP side I'll take a closer look at the ZFIN files.

**Comment by drseb** [comment]
As you can see here http://compbio.charite.de/jenkins/job/zp-owl-new/ this build hasn't been run for a while.

BTW: I am moving away from jenkins to https://github.com/Phenomics/zebrafish-phenotype-ontology and https://github.com/Phenomics/zebrafish-phenotype-ontology-build . We are working on this, but once this is finished should let you know or somebody else?

**Comment by kshefchek** [comment]
Should we be pulling the file here: https://raw.githubusercontent.com/Phenomics/zebrafish-phenotype-ontology-build/master/zp.annot_sourceinfo

For the switch, I'll make a ticket and tag you.

**Comment by drseb** [comment]
Yes, but not now. We are not finished.

**Comment by kshefchek** [comment]
It looks like ZFIN added some fields to https://zfin.org/downloads/phenoGeneCleanData_fish.txt, re-mapping the columns should fix this.

**Comment by kshefchek** [comment]
I've tracked this down to a monarch.owl - scigraph integration issue, ZP phenotypes are not being labelled as phenotypes in Neo4J.  This might be purely a ontology issue if ZP phenotypes are not subclasses of UPHENO:0001001.

For example:
https://scigraph-ontology-dev.monarchinitiative.org/scigraph/graph/ZP:0011670

**Comment by kshefchek** [comment]
This was fixed by building monarch-merged.owl with this command:

java -jar owltools http://purl.obolibrary.org/obo/upheno/monarch.owl --merge-import-closure --remove-disjoints --remove-equivalent-to-nothing-axioms -o monarch-merged.owl

and removing the axiom:
subClassOf(owl:Nothing owl:Nothing)

which comes from go-plus.owl.  Hoping this can be fixed at the go-plus level @kltm @cmungall
